doc/src/build/faucet.md

```diff
@@ -44,7 +44,7 @@ import { JsonRpcProvider, devnetConnection } from '@mysten/sui.js';
 // connect to Devnet
 const provider = new JsonRpcProvider(devnetConnection);
 // get tokens from the Devnet faucet server
-await provider.requestSuiFromFaucet(
+await provider.requestSuiFromFaucetV0(
```

Comment:
Should we include v1 in this topic as well? Or do most folks only need v0?

Reply:
V1 changes to this will follow up in a separate PR